btcec/signature.go

```diff
@@ -86,98 +74,13 @@ func (sig *Signature) IsEqual(otherSig *Signature) bool {
 }
 
 func parseSig(sigStr []byte, curve elliptic.Curve, der bool) (*Signature, error) {
```

Comment:
The paramer `der` isn't used anymore and semantically doesn't make sense as this doesn't support DER encoding anymore.